### ISSUE TITLE
If the same event occurs several time between two instants, only the last one is visible

### ISSUE THREAD

**Issue by remyzorg** [remyzorg/pendulum#18]
- improve the language with gathered signals to fix this
- tried just by changing the type of events in ocaml to lists but then it's not
possible anymore to get environnement values at anytime
- add a syntax :

```ocaml
input items (fun items x -> x :: items);
```
and

```ocaml
input (newit : inputElement Js.t)
  (onkeydown (
      fun omm ev ->
        if newit##.value##.length > 0 && ev##.keyCode = 13 then
          newit##.value :: omm else omm) [])
;
```

or even 

```ocaml
let%sync prog ?(items=fun items x -> x :: items) = ... 
```

**Comment by remyzorg** [comment]
other idea : 

```
input (newit : inputElement Js.t) {
  onkeydown = fun omm ev ->
    if newit##.value##.length > 0 && ev##.keyCode = 13 then
      newit##.value :: omm else omm) []);
};
```

**Comment by remyzorg** [comment]
This is partially fixed by commits e5ccff36e40a517d843913c70944f908113feb45 to 5514ec6b88ad772c72655b0cc271b470055c51d5 but todomvc is still not working.

The problem is the gathered event-based signal is not reset between instans, thus the the list it carries is never empty and it add all its elements everytime there is no way to access the signal and reset the value.

The only solution is to reset every geb signals at the end of each instants and maybe it's the behavior we want.
